scripts/travis/travis-tests

```diff
@@ -170,6 +170,8 @@ make -C test/others/make/ run CC="$CC"
 make -C test/others/shell-job/ run
 make -C test/others/rpc/ run
 
+mvn -f test/javaTests/pom.xml test
```

Comment:
I don't think that we need to add this test here. We have two separate travis jobs to run this test and I think these should be enough.

Reply:
I have removed the test from travis-tests.sh.